### PR TITLE
Xorg libs update

### PR DESCRIPTION
**Pull request by alarcher**
Libraries tagged after previous batch of CVEs and minor updates.

### CODE REVIEW

File: components/x11/libX11/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -64,10 +63,6 @@ link path=usr/lib/X11/locale/$(MACH64)/common/xlibi18n.so \
 link path=usr/lib/X11/locale/$(MACH64)/common/xlibi18n.so.2 \
     target=xlibi18n.so.2.0.0
 file path=usr/lib/X11/locale/$(MACH64)/common/xlibi18n.so.2.0.0
-link path=usr/lib/X11/locale/$(MACH64)/common/xlocale.so target=xlocale.so.2.0.0
-link path=usr/lib/X11/locale/$(MACH64)/common/xlocale.so.2 \
-    target=xlocale.so.2.0.0
-file path=usr/lib/X11/locale/$(MACH64)/common/xlocale.so.2.0.0
```
Comment (pyhalov) [inline components/x11/libX11/manifests/sample-manifest.p5m:70]: what did happen with xlocale?

**Comment by alarcher** [comment]
@pyhalov been running with this for a week now, any feedback?

File: components/x11/libXtst/Makefile [hunk 2]
```diff
@@ -16,24 +16,23 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=           libXtst
-COMPONENT_VERSION=        1.2.2
-COMPONENT_REVISION=       1
+COMPONENT_VERSION=        1.0.11
```
Comment (pyhalov) [inline components/x11/libXtst/Makefile]: What is this????

Comment (alarcher) [reply components/x11/libXtst/Makefile]: A missing commit????
UPDATE: non-matching number of question marks.

**Pull request merged**